ocf/alteeve/server

```diff
@@ -147,6 +149,16 @@ if ($anvil->data->{environment}{PCMK_debug})
 	$anvil->Log->level({set => 2});
 }
 
+### TODO: Use the running log level in anvil-provision-server is used to set these log levels in the pcs call.
```

Comment:
I think this is not the correct approach at all.

This env setting for this test/debug is fine, but I think it should behave very differently:

if log_level and log_secure are set via envvar (pacemaker config), then respect those ones till the end of time.

If those values are not set, attempt to get them from the DB. AFAIK the server agent already talks to the DB. If the DB connection fails, automatically switch to debug mode so we know what´s happening. Otherwise respect the DB settings.

The main rationale is that we want to be able to drive the agent logging without having to change pacemaker config. pcs calls are complex and they need to be done in different places for upgrades and stuff. There is also a corner case where we upgrade to a new version of anvil with those options, we update pacemaker config, we downgrade anvil (bug?) and things go kaboom because the old agent doesn´t support those logging options.